website/blog/2021/08/11/interview-TuZhengsong.md

```diff
@@ -1,145 +1,145 @@
 
-选择了这个 issue 之后，正松开始在 GitHub 上面开始跟社区的伙伴讨论细节上的问题，像是配置端口的 style、实现的思路等。在讨论完之后，他开始动手实现。每天晚上下班之后去翻看 Apache APISIX 的代码，测试相关用例。在经历了三四个晚上的奋斗后，测试第一次跑通了！
+After choosing this issue, Masatsu started discussing details with his community partners on GitHub, such as the style of the configuration port, the idea of the implementation, and so on. After the discussion, he started to implement it. Every night after work, he went through the Apache APISIX code and tested the use cases. After three or four nights of struggling, the test ran for the first time!
 
-当时正松内心非常的兴奋，马上按照那个新手指南的引导把 PR 提上去了。然后接受 Code Review 的意见，进一步修改代码。终于，在 10 月 18 号，PR 顺利地 merge 进了仓库。
+Masatsu was very excited and immediately followed the guide to bring up the PR. Then he accepted the comments from Code Review and further modified the code. Finally, on October 18, the PR was successfully merged into the repository.
 
 ![Apache APISIX committer tzssangglass pr](/img/blog_img/2021-08-11-2.webp)
 
-“当时我非常的激动，感觉自己终于实质性地参与到开源社区里面去了。我可以给 Apache 基金会的顶级项目提交代码了！作为一个技术工作者，这是一种技术上的自信。”
+"I was very excited and felt like I was finally substantially involved in the open source community. I could submit code to the Apache Foundation's top projects! As a technical worker, it was a technical confidence."
 
-**这是正松在开源世界的第一个 PR。**
+**This is Zhengsong's first PR in the open source world.**
 
-## 在社区印象深刻的事
+## Things that impressed me in the community
 
-> 因为 etcd 有 MaxCallRecvMsgSize 的限制，就从 etcd 的源头去解决，这是最优雅，也是最适合的。
+> Because etcd has a MaxCallRecvMsgSize limit, it was most elegant and appropriate to go to the source of etcd.
 
-当我们问到，正松在社区印象最深刻的事情是什么的时候，正松想了想说，可能是一个和 bug 有关的问题：etcd 的 MaxCallRecvMsgSize 限制。
+When we asked what was the thing that impressed Masatsu most in the community, Masatsu thought about it and said it was probably a bug-related issue: etcd's MaxCallRecvMsgSize limitation.
 
-正松曾经被 etcd 限制返回体大小的问题折磨过，社区内也多次反映、讨论过这个问题，但在 Apache APISIX 上无论怎么处理都不是很优雅。
+Masatsu had been tormented by etcd's return size limitation problem, and the community had reflected and discussed this problem many times, but it was not very elegant on Apache APISIX no matter how it was handled.
 
-直到今年 4 月份，支流科技的一位小伙伴给 etcd 提交了 PR，最终解决了这个问题。正松听说这个消息时，感到非常的震撼，当时他没有想过这个事情会以这样的方式去解决。但是现在来看的话，这个方法确实是最适合的。
+In April this year, one of our partners submitted a PR to etcd, which finally solved the problem. When Masatsu heard about this, he was very shocked, he didn't think it would be solved in such a way. But in hindsight, this was the way to go.
 
-“因为 etcd 有 MaxCallRecvMsgSize 的限制，就从 etcd 的源头去解决，这是最优雅，也是最适合的。这是很棒的跨社区合作的示例”，正松说。
+"Because etcd has a MaxCallRecvMsgSize limit, it's most elegant and appropriate to go to the source of etcd. This is great example of cross-community collaboration," says Masatsu.
```

Comment:
```suggestion
"Because etcd has a MaxCallRecvMsgSize limit, it's most elegant and appropriate to go to the source of etcd. This is great example of cross-community collaboration," says Zhengsong.
```